docs/getting-started/install/azure.md

```diff
@@ -0,0 +1,17 @@
+# Microsoft Azure
+
+- Generate an SSH key pair by executing:
+
+```
+ssh-keygen
+<enter a file name>
+<enter a password or no password for the key>
+```
+
+- Go to the [dokku on azure deployment page](https://github.com/Azure/azure-quickstart-templates/tree/master/dokku-vm) and click **Deploy to Azure**.
+
+- You'll be prompted to enter a few parameters, including a unique storage account name and a unique name for the sub-domain used for your public IP address. For the `sshKeyData` parameter, copy and paste the contents of the public key file you just created. After a few minutes the Dokku instance will be deployed.
+
+- In your browser of choice, navigate to http://[[dnsNameForPublicIP]].[[location]].cloudapp.azure.com. Where `[[dnsNameForPublicIP]]` and `[[location]]` are template parameters you used to deploy the template.
```

Comment:
Like this?:

- Go to the `[dokku on azure deployment page](https://github.com/Azure/azure-quickstart-templates/tree/master/dokku-vm)` and click **Deploy to Azure**.?